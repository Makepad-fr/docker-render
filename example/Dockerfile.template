# Use an official base image
FROM {{.BaseImage}}

# Set the exposed port
EXPOSE {{.Deployment.ExposedPort}}

# Set environment variables
{{range .Deployment.Environment}}
ENV {{.Name}}="{{.Value}}"
{{end}}

# Copy source code
COPY . {{.CopyDestination}}

# Run command
CMD {{.RunCommand}}
