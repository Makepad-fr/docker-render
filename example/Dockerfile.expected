# Use an official base image
FROM ubuntu:latest

# Set the exposed port
EXPOSE 9091

# Set environment variables

ENV DEBUG="false"

ENV LOG_LEVEL="info"


# Copy source code
COPY . /app

# Run command
CMD echo Hello World
